Box.V2/Models/BoxRetentionPolicyAssignment.cs

```diff
@@ -44,5 +45,11 @@ public class BoxRetentionPolicyAssignment : BoxEntity
         /// </summary>
         [JsonProperty(PropertyName = FieldFilterFields)]
         public virtual List<BoxMetadataFieldFilter> FilterFields { get; set; }
+
+        /// <summary>
+        /// The date the retention policy assignment begins.
```

Comment:
I found two different descriptions for this field depending on the document I looked at:

`Attribute key to look for in the source for the retention start date` 

or

`The Metadata field GUID which will be used to specify the start date for the retention policy.`

What do you think about using one of them to to accurately describe this field?